tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -2,7 +2,7 @@ ARG OS_VERSION=16.04
 FROM ubuntu:${OS_VERSION}
 
 ARG PYTHON_VERSION=3.5
-ARG OPENVINO_VERSION=2019_R3.1
+ARG OPENVINO_VERSION=2020.2
```

Comment:
can OPENVINO_VERSION be an ARG if you have INTEL_OPENVINO_DIR hardcoded to 2020.2.120,
rather than something like /data/dldt/openvino_${OPENVINO_VERSION} ?

Reply:
This wouldn't fully work, because the trailing '.120' is the serial number of the build package chosen for release, and this value could be arbitrary.
 We could use something like '/data/dldt/openvino_${OPENVINO_VERSION}.120' but it would still rely on hard coding the build number, so wouldn't really help us much.

Reply:
Never mind my previous answer, as the build number only applies to OpenVINO binary packages. In the dockerfiles for CI, we build OpenVINO from source and so are free to name the installation directory with any string. I have updated the OpenVINO installation script and docker file to use the name openvino_${OPEVINO_VERSION} that will avoid all hardcoding.